Update on publishing blogs on features that might have breaking chang… (#3007)
kubernetes/sig-release: release-team/role-handbooks/communications/README.md

Some groups you may need to contact, on top of the already mentioned SIG Docs (Blog), include:

> * SIG Contributor Experience in the `#sig-contribex` or `#sig-contribex-comms` slack channel and by attending meetings. They can help promote the feature blogs through social media. Also, see the special posts section below.

Reviewer: ```suggestion
* SIG Contributor Experience Comms Subproject in the  `#sig-contribex-comms` slack channel and by attending meetings. They can help promote the feature blogs through social media. Also, see the special posts section below.
```

* features that have been in progress for a long time and are graduating 
* features that are considered mandatory by the Release Lead.

> There have been instances where blogs requiring breaking changes for a future release have to be published. These can be published and linked in the release notes/announcement blog posts without requiring the version number in the name. 

Reviewer 2: ```suggestion
There have been instances where blogs covering breaking changes for a future release have to be published. 
There are two options for blogs covering breaking changes. They can be published as a regular feature blog (post-release communication) with the standard front matter. Alternatively, if the blog needs to go out before the release announcement, it can be published without the version prefix in the front matter and then linked in the release notes or announcement blog post.
```

Author: Sounds much better!

The Release Communications team is **NOT** responsible for social posting. [SIG Contributor Experience](https://github.com/kubernetes/community/tree/master/sig-contributor-experience) (SIG Contribex) manages the official Kubernetes social accounts and is responsible for all posts to those accounts. SIG Contribex has created automation around blog posts, so once a blog is published to the Kubernetes website, social posts are created and posted according to SIG Contribex's automation schedule. 

> If the Communications team and Release Lead determine a feature or other release communication needs a more detailed communications or calls to action, reach out to with SIG Contributor Experience for help making posts use the `@contributor-comms` tag in the `#sig-contribex` or `#sig-contribex-comms` Slack channel. 

Reviewer: I really like the direction of this, it's been a long-standing goal to have better integration between Comms and Contribex Comms. My only doubt here is the use of "or"... not that I have a great answer, but if I do not know either channels, I'm likely to be confused - should I use both? Just the first? Why? 

Perhaps something like "`#sig-contribex`  (the SIG main channel) or `#sig-contribex-comms` (the Comms subproject channel, more directly responsible for blogging and social media)", although this doesn't answer the hypothetical question I just made... 

Perhaps:

"Either channels are adequate: as a guideline, you can start in #sig-contribex and then the conversation might follow up in #sig-contribex-comms". 

@jberkus @kaslin , WDYT?

Author: The reason why I used this phrasing was because the doc only mentioned the contribex channel, but there are technically two channels to have the conversation in.

Reviewer: I've added the clarification from #sig-contribex conversations, resolving.

> If the Communications team and Release Lead determine a feature or other release communication needs a more detailed communications or calls to action, reach out to with SIG Contributor Experience for help making posts use the `@contributor-comms` tag in the `#sig-contribex` or `#sig-contribex-comms` Slack channel. 

Reviewer: ```suggestion
If the Communications team and Release Lead determine a feature or other release communication needs a more detailed communications or calls to action, reach out to with SIG Contributor Experience Comms Subproject for help making posts use the `@contributor-comms` tag in the `#sig-contribex-comms` Slack channel. 
```
